src/components/smart_objects/include/smart_objects/smart_schema.h

```diff
@@ -68,10 +68,11 @@ class CSmartSchema FINAL {
    * @param MessageVersion to check mobile RPC version against RPC Spec History
    * @return ns_smart_objects::errors::eType
```

Comment:
Could add a description for the new `allow_unknown_enums` param to the function brief